networkx/generators/geometric.py

```diff
@@ -279,14 +432,19 @@ def geographical_threshold_graph(n, theta, alpha=2, dim=2, pos=None,
     nx.set_node_attributes(G, 'weight', weight)
     nx.set_node_attributes(G, 'pos', pos)
 
+    #if p_dist not supplied, use default r^alpha
+    if p_dist == None:
+        def p_dist(r):
+            return r**-alpha
+
```

Comment:
Maybe the best thing to do is *break* backwards compatibility here completely.  That way we can clearly document the alpha -> -alpha change by showing how to update existing code.